docs/guide/contributing.md

```diff
@@ -12,7 +12,8 @@ title: 'Contributing'
 4. Type in `npm link` which creates a symlink and thereby allowing `teachcode` to be accessed globally.
 5. Make your life changing changes.
 6. Ensure that your code is free from linting errors and is as per prettier conventions by typing in `npm run lint`.
-7. Now you may push it to the remote repository by switching over to another branch and finally proposing a Pull Request to the base fork.
+7. We use [commitlint conventional naming rules](https://www.npmjs.com/package/@commitlint/config-conventional#rules) for our commits, make sure to follow them.
+8. Now you may push it to the remote repository by switching over to another branch and finally proposing a Pull Request to the base fork.
 
```

Comment:
I guess we can also have a mention regarding the branch naming conventions, such as `feat`, `hotfix`, 'docs' prefixes for feature proposals, bug fixes, and docs-related respectively.

Reply:
You may follow the approach as [here](https://github.com/madlabsinc/mevn-cli#how-should-i-write-a-commit-message).

Reply:
Just realized there was already some mention to branch naming in the same file, so I expanded the description a bit.

Reply:
Do you also want the commit message pattern described in `contributing.md`?

Reply:
Just have a mention of it within `README.md` following [this](https://github.com/madlabsinc/mevn-cli#how-should-i-write-a-commit-message) approach.

Reply:
Alright, I've updated this project's `README.md` file to include two new sections under "Contributing": "How should I write a commit message?" and "How should I name my branches?".